tests/e2e.test.ts

```diff
@@ -384,5 +385,45 @@ describe('End-to-end tests', () => {
         `)
       })
     })
+    describe('--error-format option', () => {
+      it('fails with an invalid error format', async () => {
+        const invalidErrorFormat: string = 'invalid-format'
+
+        const checkOutput = await cli(
+          `check --error-format ${invalidErrorFormat}`,
+          basicTsErrorOutput
+        )
+        expect(checkOutput.code).toBe(1)
+        expect(checkOutput.stderr).toMatch(
+          `error: option '--error-format [error-format]' argument '${invalidErrorFormat}' is invalid.`
+        )
+      })
```

Comment:
Should we add a positive tests for all options in the enum here? 

I mean like this:

```javascript
it.each(Objects.key(ErrorFormat)('succeed with valid error format', (errorFormat) => {
        const checkOutput = await cli(
          `check --error-format ${invalidErrorFormat}`
        )
        expect(checkOutput.code).toBe(0)
})
```

or something like that?

Reply:
I modified your example and implemented it.